includes/polling/ports.inc.php

```diff
@@ -227,7 +227,7 @@
 if ($config['enable_ports_etherlike']) {
     echo 'dot3Stats ';
     $port_stats = snmpwalk_cache_oid($device, 'dot3StatsEntry', $port_stats, 'EtherLike-MIB');
-} else {
+} elseif ($device['os'] != 'asa') {
```

Comment:
@murrant what's your thoughts on this. It can be extended out to other poller modules where we do checks for varying OS' to include / exclude stuff to poll.